README.md

```diff
@@ -135,12 +135,21 @@ All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metapa
 
 - DSC Composite Resources are not supported.
 - PSObjects like 'PSCredential' as parameters value not yet supported.
+- DSC requires PowerShell "Execution Policy" to be set appropriately for "LocalMachine" scope. If you see the error below, see [MODULES-2500](https://tickets.puppetlabs.com/browse/MODULES-2500) for more information.
+
+~~~
+Error: /Stage[main]/Main/Dsc_xgroup[testgroup]: Could not evaluate: Importing module MSFT_xGroupResource failed with
+ error - File C:\Program
+Files\WindowsPowerShell\Modules\PuppetVendoredModules\xPSDesiredStateConfiguration\DscResources\MSFT_xGroupR
+esource\MSFT_xGroupResource.psm1 cannot be loaded because running scripts is disabled on this system. For more
+information, see about_Execution_Policies at http://go.microsoft.com/fwlink/?LinkID=135170.
+~~~
```

Comment:
@ferventcoder Maybe we should make it a prerequisite for the module to have the PowerShell execution policy be set to something reasonable.